packages/react-docs/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "analyze": "yarn build:docs --analyze",
     "build:docs": "pf-docs-framework build all",
-    "clean": "rimraf .cache public static/assets static/base.css src/generated/**/*.js",
+    "clean": "rimraf .cache public static/assets static/base.css src/generated/**/*.js node_modules",
```

Comment:
When using a local version of the documentation-framework, it helps to clear out these node_modules, as sometimes not doing so leads to false-positives/negatives.